test/powershell/Modules/Microsoft.PowerShell.Archive/Pester.Commands.Cmdlets.Archive.Tests.ps1

```diff
@@ -12,9 +12,8 @@ Describe "Test suite for Microsoft.PowerShell.Archive module" -Tags "CI" {
     }
     BeforeAll {
         # remove the archive module forcefully, to be sure we get the correct version
-        if ( Get-Module Microsoft.PowerShell.Archive. ) {
-            Remove-Module Microsoft.PowerShell.Archive -force
-        }
+        Remove-Module Microsoft.PowerShell.Archive -force
```

Comment:
The `Get-Module` call is to avoid getting the error message `Remove-Module : No modules were removed. Verify that the specification of modules to remove is correct and those
modules exist in the runspace.` because Archive module is only loaded when it's used.